src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -572,4 +587,72 @@ void testDeleteInventory() throws Exception {
 		assertThat(deletedInventory).isNotNull();
 		assertThat(deletedInventory.getStatus()).isEqualTo(InventoryStatus.canceled.toString());
 	}
+
+	@Test
+	void testValidateMedicalInventory() throws Exception {
```

Comment:
If you check the coverage of this method, you will see that most of it is not covered:

![image](https://github.com/user-attachments/assets/14d7206f-684d-47d8-a43a-dfbe467f19cf)